source/modules/MEI.shared.xml

```diff
@@ -8387,20 +8390,22 @@
       </attDef>
     </attList>
     <remarks xml:lang="en">
-      <p>The <att>type</att> attribute may be used to classify the title according to some
-        convenient typology. Sample values include: main (main title), subordinate (subtitle, title
-        of part), abbreviated (abbreviated form of title), alternative (alternate title by which the
-        work is also known), translated (translated form of title), uniform (collective title), and
-        desc (descriptive title). The <att>type</att> attribute is provided for convenience in
-        analysing titles and processing them according to their type; where such specialized
-        processing is not necessary, there is no need for such analysis, and the entire title,
-        including subtitles and any parallel titles, may be enclosed within a single <gi
-        scheme="MEI">title</gi> element. Title parts may be encoded in <gi scheme="MEI"
-        >titlePart</gi> sub-elements. The name of the list from which a controlled value is taken
-        may be recorded using the <att>auth</att> attribute. The number of initial characters (such
-        as those constituting an article or preposition) that should not be used for sorting a title
-        or name may be indicated in the <att>nonfiling</att> attribute.</p>
-    </remarks>
+  <p>The <att>type</att> attribute may be used to classify the title according to some
+    convenient typology. Sample values include: main (main title), subordinate (subtitle,
+    title of part), abbreviated (abbreviated form of title), alternative (alternate title
+    by which the work is also known), translated (translated form of title), uniform
+    (collective title), parallel (title in another language or script appearing with the
+    main title), and desc (descriptive title). The <att>type</att> attribute is provided
+    for convenience in analysing titles and processing them according to their type; where
+    such specialized processing is not necessary, there is no need for such analysis, and
+    the entire title, including subtitles and any parallel titles, may be enclosed within
+    a single <gi scheme="MEI">title</gi> element. Title parts may be encoded in <gi
+      scheme="MEI">titlePart</gi> sub-elements. The name of the list from which a
+    controlled value is taken may be recorded using the <att>auth</att> attribute. The
+    number of initial characters (such as those constituting an article or preposition)
+    that should not be used for sorting a title or name may be indicated in the
+      <att>nonfiling</att> attribute.</p>
+</remarks>
```

Comment:
I think the formatting and indentation is off here.

Reply:
It looks ok to me, but I can't figure out how to see the change in the context of the entire file.  I also don't know how to revise the PR.

Reply:
This should work to see it? https://github.com/music-encoding/music-encoding/blob/949464601e80914856d3467a6132e061b7e29c51/source/modules/MEI.shared.xml#L8392-L8408

Reply:
To change it you should have a local branch, `develop-title-type`, on your computer. If so, you can change to that branch, fix it, then commit and push. The updates should appear automatically.

Reply:
I think I fixed the indentation.  Have a look-see now?

Reply:
Indentation looks good to me. (Not sure if others here will want the `</p>` on a new line or not, but I'm not bothered.)

There's also an outstanding suggestion for using a capital "T" to start -- you can just click "commit suggestion" above if you agree with it.